doc/API.md

```diff
@@ -18,11 +18,13 @@ Display general information about the current daemon state.
 
 #### Response
 
-| Field         | Type    | Description                                                   |
-| ------------- | ------- | ------------------------------------------------------------- |
-| `blockheight` | integer | Current block height                                          |
-| `network`     | string  | Answer can be `mainnet`, `testnet`, `regtest`                 |
-| `version`     | string  | Version following the [SimVer](http://www.simver.org/) format |
+| Field              | Type    | Description                                                          |
+| ------------------ | ------- | -------------------------------------------------------------------- |
+| `blockheight`      | integer | Current block height                                                 |
+| `network`          | string  | Answer can be `mainnet`, `testnet`, `regtest`                        |
+| `synced`           | bool    | True if the daemon is synced with the blockchain                     |
+| `syncing_progress` | integer | The syncing progress as percent -- only present if `synced` is false |
```

Comment:
This seems like a duplicate to me. What about 
```suggestion
| `sync` | integer | The synchronization progress as a percentage. |
```
`synced is true == sync equals to 1`.